src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_reader.py

```diff
@@ -78,19 +79,22 @@ def _read_data(self) -> pd.DataFrame:
             errors="coerce",
         )
         file_data = file_data.dropna(subset=DATE_HEADER)
-
-        return file_data
+        return [SeriesData(x) for _, x in file_data.iterrows()]
```

Comment:
Note: this change is for compatibility with text read changes, it's just converting to `SeriesData` earlier, so I don't have to bother changing the series below back to a dataframe.